From 8b08e19e21d0dfdaa364ed732993668e20d7ed36 Mon Sep 17 00:00:00 2001
From: Sagi Maimon <smaimon@habana.ai>
Date: Tue, 19 Jan 2021 16:56:17 +0200
Subject: [PATCH] [SW-28838] hl: disable linux console on secure mode

On secure mode, we want to prevent bmc console access.

Change-Id: Ie8bf8153095c3810cce6b4f35806620827c38f3d
Signed-off-by: Sagi Maimon <smaimon@habana.ai>
---

diff --git a/board/habanalabs/gaudi/uboot_asic.fragment b/board/habanalabs/gaudi/uboot_asic.fragment
index 0e46451..83a696a 100644
--- a/board/habanalabs/gaudi/uboot_asic.fragment
+++ b/board/habanalabs/gaudi/uboot_asic.fragment
@@ -9,3 +9,4 @@
 CONFIG_SYS_EEPROM_PAGE_WRITE_DELAY_MS=5
 CONFIG_SYS_I2C_EEPROM_ADDR=0x54
 CONFIG_SYS_I2C_EEPROM_ADDR_LEN=1
+CONFIG_DISABLE_CONSOLE=y
